ui/component/or-mwc-components/src/or-mwc-input.ts

```diff
@@ -678,6 +678,10 @@ export class OrMwcInput extends LitElement {
     @property({type: Boolean})
     public required: boolean = false;
 
+    /* Triggers OrInputChangedEvent based on @input instead of @change. Only supported for text-field input. */
+    @property({type: Boolean})
+    public live: boolean = false;
+
```

Comment:
Can't any consumer just attach an input listener from outside of the `or-mwc-input` component to do something when input occurs?

Reply:
You are indeed correct. I thought that event bubbling wasn't working for `or-mwc-input`, but it apparently is.
I removed it from `or-mwc-input` and added a `@input` listener on the consumer component instead.

Reply:
Event bubbling was not working for `@input` listener unfortunately.

Since `@input` only dispatches the keypress, for example `H`, we have no idea what its current value is.
Normally we would use `event.currentTarget` for this, but with the shadow DOM it won't return the correct element.
Directly referencing `OrMwcInput.value` also won't work, as this value isn't updated before a user submit.

I think adding a `live` property, where `@input` behaves exactly the same as `@change` does now,
is the most useful solution here. (instead of simply redispatching the input event)